src/utils/detailPages.ts

```diff
@@ -19,4 +20,20 @@ export const getDetailPagePaths = (async (config: any, model: Models) => {
   }
 
   return routes;
-});
\ No newline at end of file
+});
+
+export const getCoverImage = (record: any) => {
+  if (record?.relatedRecords?.mediaContents && record.relatedRecords.mediaContents.length > 0) {
+    return record.relatedRecords.mediaContents[0].content_url
```

Comment:
The `content_url` property will load the image from Core Data's Active Storage backend. We'll probably want to load it using the IIIF Image API, for which a few different properties should be available:

`content_iiif_url` - Full image
`content_preview_url` - Max width of 500px
`content_thumbnail_url` - Max width of 250px

Reply:
I've replaced `content_url` with `content_iiif_url`.